setup.py

```diff
@@ -114,22 +114,27 @@ def localversion_func(version: ScmVersion) -> str:
         ("pyyaml>=6.0.1,<=6.0.3" if BUILD_TYPE == "release" else "pyyaml>=6.0.1"),
         # librosa dependency numba is currently not compatible with numpy>=2.3
         # https://numba.readthedocs.io/en/stable/user/installing.html#version-support-information
-        ("numpy>=2.0.0,<=2.3.3" if BUILD_TYPE == "release" else "numpy>=2.0.0"),
+        ("numpy>=2.0.0,<=2.3.5" if BUILD_TYPE == "release" else "numpy>=2.0.0"),
         (
             "requests>=2.32.2,<=2.32.5"
             if BUILD_TYPE == "release"
             else "requests>=2.32.2"
         ),
         ("tqdm>=4.66.3,<=4.67.1" if BUILD_TYPE == "release" else "tqdm>=4.66.3"),
-        ("torch>=2.7.0,<=2.8.0" if BUILD_TYPE == "release" else "torch>=2.7.0"),
+        ("torch>=2.7.0,<=2.9.1" if BUILD_TYPE == "release" else "torch>=2.7.0"),
         (
-            "transformers>=4.54.0,<=4.56.2"
+            "transformers>=4.54.0,<=4.57.3"
             if BUILD_TYPE == "release"
-            else "transformers>=4.54.0"
+            else "transformers>=4.54.0,<=4.57.3"
         ),
-        ("datasets>=4.0.0,<=4.1.1" if BUILD_TYPE == "release" else "datasets>=4.0.0"),
+        ("datasets>=4.0.0,<=4.4.1" if BUILD_TYPE == "release" else "datasets>=4.0.0"),
         (
-            "accelerate>=1.6.0,<=1.10.1"
+            "auto-round>=0.9.1,<=0.9.2"
+            if BUILD_TYPE == "release"
+            else "auto-round>=0.9.1"
```

Comment:
Will the autoround lower bound work with our accelerate dependency?

Reply:
Oh good catch, yes the 0.9.1 will only work with `accelerate` 1.10.0. Should we only allow `auto-round >=0.9.2`?

Reply:
Thanks, updated.